### DISCUSSION TITLE
Do we want Edgeshark integration ?

### DISCUSSION THREAD

**Discussion by DanPartelly** [ipspace/netlab#1423] [Ideas]
Do we want Edgeshark integration in external tools , like there is for graphite ? For full functionality the user would have to install the wireshark capture plugins on their own.

**Comment by ipspace** [comment]
Sure. Will you figure out the fundamentals and I can jump in if we need something specific (like the challenges of doing packet capture)?

**Reply by DanPartelly** [reply]
yes, of course. This is why I asked. need something netlab related to work on weekends and  which is useful to users. 

> Sure. Will you figure out the fundamentals and I can jump in if we need something specific (like the challenges of doing packet capture)?

**Reply by ipspace** [reply]
> need something netlab related to work on weekends and which is useful to users.

Wouldn't happen to be a Junos expert, would you? 😉

**Reply by DanPartelly** [reply]
No, I am not. If I can help with something ill gladly doit, but Im not the expert you are looking for =) 
> Wouldn't happen to be a Junos expert, would you? 😉

**Comment by ipspace** [comment]
Implemented
